serverlessworkflow/modules/ROOT/pages/cloud/quarkus/kubernetes-service-discovery.adoc

```diff
@@ -212,7 +212,7 @@ For example, consider an application that consumes a resource running on Kuberne
 .Example URI
 [source,shell]
 ----
-org.kie.kogito.sw.knative.service=knative:services.v1/serverless-workflow-greeting-quarkus/greeting-quarkus-cli
+org.kie.kogito.sw.knative.service=${knative:services.v1/serverless-workflow-greeting-quarkus/greeting-quarkus-cli}
```

Comment:
Maybe add also some suffix like `/endpoint` to showcase the new feature more?

Reply:
@MarianMacik not sure @hbelmiro gonna update that one. Could you propose the change you mean ?

Reply:
Updated. Thank you @MarianMacik.

Reply:
thank you @hbelmiro